OpenSim/Tests/Wrapping/testWrapCylinder.cpp

```diff
@@ -57,6 +69,51 @@ namespace {
             "PathSegment{start: " << path.start << ", end: " << path.end << "}";
     }
 
+    PathSegment operator*(
+        const SimTK::Rotation& rot,
+        const PathSegment& path)
+    {
+        return PathSegment{
+            rot * path.start,
+            rot * path.end,
+        };
+    }
+
+    // Angular distance from start- to end-angle in either positive or negative direction.
+    double AngularDistance(
+        double startAngle,
+        double endAngle,
+        bool positiveRotation)
+    {
+        double distance = std::fmod(endAngle - startAngle, c_TAU);
+        while ((distance < 0.) && positiveRotation) {
+            distance += c_TAU;
+        }
+        while ((distance > 0.) && !positiveRotation) {
+            distance -= c_TAU;
+        }
+        return distance;
+    }
+
+    // Returns direction of shortest angular distance for a vector aligned with the
+    // start point to become aligned with the end point (true if positive).
+    bool DirectionOfShortestAngularDistance(
+        SimTK::Vec2 start,
+        SimTK::Vec2 end)
+    {
+        return AngularDistance(
```

Comment:
Current formatting having a trailing `<=` is undesirable. Reformat to `double dist = f(...); return dist <= SimTK::Pi`